mlflow/entities/model_registry/model_version.py

```diff
@@ -1,9 +1,16 @@
 from mlflow.entities.model_registry import RegisteredModel
 from mlflow.entities.model_registry._model_registry_entity import _ModelRegistryEntity
 from mlflow.protos.model_registry_pb2 import ModelVersion as ProtoModelVersion
+from mlflow.utils import experimental
 
 
 class ModelVersion(_ModelRegistryEntity):
+    """
+    Note:: Experimental: This entity may change or be removed in a future release without warning.
```

Comment:
Cannot use `@experimental` annotation for classes since python 2 does not allow `__doc__` attributed to be edited.

```
AttributeError: attribute '__doc__' of 'type' objects is not writable
```